kafka/common/pkg/kafka/consumer_handler_test.go

```diff
@@ -42,6 +42,10 @@ type mockConsumerGroupSession struct {
 	marked bool
 }
 
+func (m *mockConsumerGroupSession) Commit() {
+	panic("implement me")
```

Comment:
@pierDipi was that introduced on 1.27.x APIs ?

Reply:
Yes, they added the ability to [manually committing offsets](https://github.com/Shopify/sarama/pull/1699). `Commit` performs a blocking synchronous operation.